tests/assets/kubernetes_logs.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
```

Comment:
a short description could be useful here

Reply:
Good point, this was just a copy from provider so I didn't think of touching this files